tests/framework/integration/testing.go

```diff
@@ -84,6 +99,21 @@ func BeforeTest(t testutil.TB, opts ...TestOption) {
 		testutil.RegisterLeakDetection(t)
 	}
 
+	if options.failpoint != nil && len(options.failpoint.name) != 0 {
+		err := gofail.Enable(options.failpoint.name, options.failpoint.payload)
+		switch {
+		case err == gofail.ErrNoExist:
```

Comment:
```suggestion
		case errors.Is(err, gofail.ErrNoExist):
```

Reply:
Yeah, that's the best practice of golang error comparing.. Thanks for pointing it out.